package.json

```diff
@@ -52,7 +52,9 @@
   },
   "dependencies": {
     "bulk-data-client": "git+https://git@github.com/smart-on-fhir/bulk-data-client",
+    "colors": "^1.4.0",
     "commander": "^9.4.1",
+    "prompt-sync": "^4.2.0",
```

Comment:
This library hasn't been updated since 2019 and an [issue on it's GitHub page indicates there are unaddressed security vulnerabilities](https://github.com/heapwolf/prompt-sync/issues/58).

The library that poster recommended is also now archived. Based on the latest commits from right before the archive, I'm guessing it's because `Now, ECMAScript supports Promise, await and async.`

[inquirer](https://www.npmjs.com/package/inquirer) seems to be well maintained based on existing GitHub issues and has significiant weekly downloads. (I found it via [this Digital Ocean article](https://www.digitalocean.com/community/tutorials/nodejs-interactive-command-line-prompts)).

I'd suggest either:

1. Writing this with the [built in node.js readline](https://nodejs.org/api/readline.html)
2. Using a maintained library like [inquirer](https://www.npmjs.com/package/inquirer)

What do you think?

Reply:
Thanks for doing some research into this - I had used `prompt-sync` to be consistent with what is used in the SMART client, but having a well maintained dependency seems more important to me. I will look into these options!

Reply:
I updated this PR to instead use the built-in node.js readline 👍